CONTRIBUTING.md

```diff
@@ -1,7 +1,122 @@
+
+### 1) Submit coding guideline issue
+
+For a new coding guideline you'd like to contribute, start with opening a [coding guideline issue](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/new?template=CODING-GUIDELINE.yml).
+
+#### 1.1) Finding the FLS ID
+
+Note that the FLS ID should be filled according to the FLS paragraph ID for which the guideline is covering. One way to go about finding this is to inspect the page using your web browser. You'll be looking for something like:
+
+
+```html
+<p><span class="spec-paragraph-id" id="fls_4rhjpdu4zfqj">4.1:1</span>
+```
+
+You would then pull `fls_4rhjpdu4zfqj` to place in the FLS ID field.
+
+### 2) Create a Draft with a Member
+
+A member of the Coding Guidelines Subcommittee should get you a first review with some feedback within 14 days of submission. You'll work with one or more members to flesh out the concept and ensure the guideline is well prepared.
+
+### 3) A Pull Request is generated
+
+Once an issue has been well-developed enough, a subcommittee member will mark the issue with the label `sign-off: create pr from issue` to generate a pull request. You will see a GitHub Workflow trigger and a pull request will be created momentarily.
+
+### 4) Iterate on Feedback
+
+The generated Pull Request may attract additional feedback or simply be an easier place to suggest targeted edits.
+
+As the contributor of the coding guideline and opener of the issue, you'll respond to comments, discuss, all the normal things on the pull request.
+
+#### 4.1) Apply changes to the Guideline's Issue
+
+If you agree with the suggested changes, then rather than making changes on the opened pull request, you will return to the original issue you opened via the coding guideline issue template, and make the updates there.
+
+#### 4.2) Re-generate the Pull Request
+
+When you have completed all feedback given to you, ping one of the subcommittee members. They will then remove and affix the label `sign-off: create pr from issue` to push the changes made in the issue to the already opened pull request.
+
+### 5) Your Guideline gets merged
+
+Once the coding guideline contents have passed review, a subcommittee member will approve the pull request, and put it on the merge queue to be merged.
+
+### You just contributed a coding guideline!
+
+That's it!
+
+
+## Writing a guideline locally (less typical, not recommended)
+
+While it is possible to create guidelines locally and open pull requests yourself, we encourage contributors to make use of the process described above since it handled some of the fiddly details for you as a guideline writer.
+
+Generally speaking, pull requests for guidelines which do not follow the issue to pull request workflow described above will be closed with a recommendation to follow the workflow.
```

Comment:
If we don't want people to do this it might be better not to even mention it here?

Reply:
For sure, but I would tend to say that this doesn't really help. If you need to create it locally do that. But most folks won't and shouldn't, so we don't need to paragraphs of text to say that (unless we find ourselves inundated with PR....)

You could add a single line in step one open issue saying if (and only if) you have formatting that doesn't work, open a PR locally...